---
title: "Identification of genotype-phenotype associations in Phelan-McDermid Syndrome using patient-sourced data from an international registry"
keep_md: true
output:
  pdf_document:
    template: template.tex
    keep_tex: true
    latex_engine: xelatex
  word_document:
    reference_docx: template.docx
bibliography: PMS_DN_biblatex.bib
csl: humangenet.csl
author: Maxime Wack, Claire Hassen-Khodja, Michael T. McDuffie, Geraldine Bliss, Liz Horn, Andria Cornell, Cartik Kothari, Catalina Betancur, Rebecca C. Davis, Megan O'Boyle, Isaac Kohane, Paul Avillach
abstract: |
  # **Abstract**
  Phelan-McDermid Syndrome (PMS) is a syndromic form of autism caused by terminal deletions in the long arm of chromosome 22. These affect the SHANK3 gene and variably associate with autism, global developmental delay, delayed speech, neonatal hypotonia, and mild dysmorphic features. Further confirming the role of SHANK3 in PMS, isolated haploinsufficiency of the gene has been linked to of a subset of PMS features. We utilized the PMS International Registry (PMSIR), which compiles clinical data in the form of patient-reported outcomes (PRO), as well as patient-sourced genetic test results. Data from the PMSIR had been harmonized and integrated into the i2b2/tranSMART clinical & genomics data warehouse. For this study, we conducted genotype-phenotype analyses using regression models with deletion size as a potential predictor of clinical outcomes. We gathered data from `r article$nb_pat$included` who either had deletions, ranging from `r round(article$genetics$range["Min."]/1000, digits = 3)` kb to `r round(article$genetics$range["Max."]/10^6, digits = 3)` Mb, or small isolated SHANK3 mutations (`r article$nb_pat$mut_sel` patients). We showed that increased deletion size was significantly associated with delay in gross motor acquisitions, renal malformations, mild dysmorphic features (large fleshy hands, dysplastic toenails and sacral dimple), and a spectrum of conditions related to poor muscle tone. These results suggest that genes upstream of SHANK3 also play a role in PMS. Overall, we demonstrated the feasibility of using data from an entirely patient-sourced registry (PROs and curated genetic test reports) to conduct genotype-phenotype analyses. We both replicated previous results and presented novel findings to verify the ability of registry data to uncover new associations between comorbidities and deleted chromosomal regions in PMS.
---

```{r load, echo = F, message = F}
library(rmarkdown)
library(knitr)
library(dplyr)
library(pander)
library(stringr)
library(desctable)

readRDS("article.rds") -> article

pvalue <- function(var)
{
  article$results_ranges %>%
    filter(text == var) %>%
    .$p.adj %>%
    prettyNum(digits = 2)
}

or <- function(var)
{
  article$results_ranges %>%
    filter(text == var) %>%
    mutate(or = or %>% prettyNum(digits = 2)) %>%
    transmute(or = str_c(or, " ", IC_OR)) %>%
    pull(or)
}

phenotypes <- function(text)
{
  text %>%
    str_c(collapse = ", ")
}

range_p_or <- function(p_or)
{
  p_or %>%
  range %>%
    prettyNum(digits = 2) %>%
    str_c(collapse = " to ")
}

result_for_group <- function(group)
{
  article$results_ranges %>%
    filter(Group == group,
           p.adj < .05) %>%
  arrange(p.adj) -> res

  str_c("(",
        phenotypes(res$text),
        " (*p values* from ",
        range_p_or(res$p.adj),
        ", *ORs* from ",
        range_p_or(res$or),
        "))")
}

opts_chunk$set(echo = F,
               message = F,
               error = F,
               warning = F,
               fig.width = 12,
               fig.height = 7)
```

# Introduction

The 22q13 deletion syndrome (OMIM 606232), also called Phelan-McDermid Syndrome (PMS) [@phelan_phelan-mcdermid_1993 ; @phelan_22q13.3_2012], is a syndromic form of Autism Spectrum Disorder (ASD).[@phelan_22q13.3_2012]
Approximately 2,200 cases have been diagnosed worldwide, 559 of which have been confirmed in the US and registered with the Phelan-McDermid Syndrome Foundation (PMSF).
No treatment are yet available, although one drug candidate, IGF1, has used successfully in a phase 2 randomized controlled trial.[@kolevzon_pilot_2014]

Clinically, PMS patients present with autistic features and autism or autistic-like behavior at a rate ranging from 44% to 94% of patients.[@kolevzon_phelan-mcdermid_2014 ; @costales_phelanmcdermid_2015]
An estimated 84% of cases meet the criteria for autism spectrum disorder (ASD), based on the Autism Diagnostic Observation Schedule, a gold standard of diagnosis.[@soorya_prospective_2013]
More than 98% of patients suffer global developmental delay,[@phelan_22q13.3_2012 ; @phelan_deletion_2008] and nearly 100% show intellectual disability (ID) with delayed or absent speech.[@phelan_22q13.3_2012 ; @kolevzon_phelan-mcdermid_2014 ; @costales_phelanmcdermid_2015 ; @soorya_prospective_2013 ; @phelan_deletion_2008 ; @durand_mutations_2007 ; @leblond_meta-analysis_2014]
Overall, PMS accounts for an estimated 0.5% of ASD cases [@kolevzon_phelan-mcdermid_2014 ; @costales_phelanmcdermid_2015 ; @leblond_meta-analysis_2014 ; @gong_high_2012 ; @betancur_shank3_2013] and 2% of ID cases,[@kolevzon_phelan-mcdermid_2014 ; @leblond_meta-analysis_2014 ; @gong_high_2012 @cooper_copy_2011] with an equal number of male and female patients.[@costales_phelanmcdermid_2015 ; @phelan_deletion_2008]
In addition, PMS patients are characterized by specific behavioral features, such as impairments in communication and social activities, chewing non-food items (80-90%), teeth grinding (~25%), and aggressive behavior (10-15%).[@phelan_phelan-mcdermid_1993 ; @phelan_22q13.3_2012 ; @kolevzon_phelan-mcdermid_2014 ; @phelan_deletion_2008 ; @sarasua_clinical_2014]

From a genetic perspective, PMS is caused by random deletions in the distal long arm of chromosome 22.
These remove a gene, called *SHANK3*——for SH3 and multiple ankyrin repeat domain3——which is located at the chromosome's end.
The disease is diagnosed by genetic testing, which identifies either a heterozygous deletion at chromosome 22q13.3 (with involvment of at least part of *SHANK3*) or a heterozygous pathogenic variant in *SHANK3*.
A potential treatment, IGF1, as been shown to restore the *SHANK3* pathway.[@shcheglovitov_shank3_2013]

*SHANK3* encodes a post-synaptic scaffolding protein that binds to several different neurotransmitter receptors.
It is thought to be play a critical role in the assembly and plasticity of the postsynaptic density (PSD) in brain synapses.
Researchers have also demonstrated *SHANK3* involvment in non-PMS autistic spectrum disorders,[@durand_mutations_2007 ; @leblond_meta-analysis_2014 ; @betancur_shank3_2013 ; @shao_commonly_2014 ; @moessner_contribution_2007 ; @boccuto_prevalence_2013 ; @uchino_shank3_2013 ; @herbert_shank3_2011] further indicating its role in the neurobehavioral features of PMS.

Deficiency (deletion or loss of function) of a single copy of *SHANK3* has been shown to be sufficient to cause the disease,[@wilson_molecular_2003] in both patient cases and murine models that involve *de novo* point mutations or interstitial deletions that exclusively disrupt *SHANK3*.[@durand_mutations_2007 ; @moessner_contribution_2007 ; @bozdagi_haploinsufficiency_2010 ; @macedoni-luksic_deletion_2013 ; @bonaglia_disruption_2001 ; @delahaye_chromosome_2009]
PMS patients with isolated *SHANK3* mutations or deletions show major speech delays and profound ID, global developmental delay, ASD symptoms, slightly delayed motor milestones, mild hypotonia, and minor dysmorphic features (dolichocephaly, epicanthic folds).
*SHANK3* haploinsufficiency and Fragile X Syndrome are the two most frequent single gene genetic causes of autism/autism spectrum disorder.

PMS patients exhibit heterogeneous chromosomal abnormalities in the 22q13 region: terminal deletions, interstitial deletions, duplications and ring chromosome.
These result in an overall population of patients with randomly distributed deletions, ranging in size from 100 kb to over 9 Mb.[@phelan_deletion_2008]
Previous studies have shown a gradient in the severity of cognitive impairments.[@leblond_meta-analysis_2014]
These are associated with the size of the deletions and linked to the presence of other phenotypic features.[@soorya_prospective_2013 ; @sarasua_clinical_2014 ; @sarasua_association_2011 ; @sarasua_22q13.2q13.32_2014]

At the same time, a growing number of cases involve symptoms consistent with a diagnosis of 22q13 deletion syndrome (severe developmental delays, hypotonia with feeding difficulties, overgrowth, speech impairments, seizures, abnormal neuroimaging, …) but in patients lacking a classical terminal 22q deletion.
More intriguing, the non-classical cases present interstitial deletions as small as 0.72Mb beyond the locus of *SHANK3*.
This indicates that a subset of features of PMS are not caused solely by *SHANK3* haploinsufficiency.[@fujita_girl_2000 ; @wilson_interstitial_2008 ; @simenson_patient_2014 ; @vittoria_disciglio_interstitial_2014 ; @maitz_expanding_2008]

To explore this phenomenon more closely, we utilized the Phelan-McDermid Syndrome International Registry (PMSIR),[@PMSIR_phelan-mcdermid_2011] a patient/family-driven registry, founded by the parents of PMS patients and launched in 2011 by the Phelan-McDermid Syndrome Foundation (PMSF).
The only international registry for PMS, the registry today includes 1,198 of the 2,200 known patients.
Its phenotypic and genotypic information is annotated by the patient families.

We hypothesized that while retrospective and not recorded in the context of a clinic, still patient-reported data (including genetic test results obtained from the Phelan-McDermid Syndrome International Registry)[@PMSIR_phelan-mcdermid_2011] could be used to conduct deletion analyses and uncover phenotypes and comorbidities significantly linked to genes other than *SHANK3*.
Our results have implications not only for PMS but also for those interested in using patient-reported data to shed new light on other rare diseases.

# Methods

Funded by the US Patient Centered Outcome Research Institute (PCORI), the Phelan-McDermid Syndrome Data Network (PMS_DN) project aims to advance knowledge of PMS and related conditions by integrating patient-reported outcomes, curated genetic test results, and knowledge extracted from clinical notes in the centralized i2b2/tranSMART platform to facilitate patient-centered research.
The clinical notes were not used in this study.
The network promotes a culture of transparency, as well as authentic engagement and leadership of families in the network's vision, governance, and generation of research priorities.
PMS_DN uses data entered onto the online Phelan-McDermid Syndrome International Registry by patients families.
The registry is comprised of 1,300 patient-reported outcome (PRO) items over three distinct questionnaires: a “clinical” questionnaire, based on diagnosed comorbidities, symptoms, tests, and treatments for the entire range of known pathologies and features associated with PMS; a “developmental” questionnaire, focusing on physical, motor, behavioral, cognitive, and social development; and an “adult” questionnaire aimed at patients aged 12 or older, regarding the evolution of symptoms after puberty.
(The latter was not used in this study.)
Patient families can retake the surveys any number of times, allowing for longitudinal analysis and ensuring the most complete and accurate information.

Alongside phenotypic data, the registry also contains curated genetic testing of chromosomal abnormalities from karyotypes, FISH probes, comparative genomic hybridization (aCGH), microarrays, and sequencing results represented using ISCN Standard Nomenclature.[@international_standing_committee_on_human_cytogenetic_nomenclature_iscn_2013]
The scanned paper reports of genetic test results are uploaded by families onto the platform, then manually curated by trained genetic counselors, who fill 57 structured fields documenting genetic abnormalities.

To facilitate the continuous updating of the PMS registry, all phenotypic and genotypic data were integrated in the i2b2/tranSMART[@athey_transmart:_2013 ; @canuel_translational_2015 ; @murphy_serving_2010] data warehouse.
In addition, a PMS ontology was created to tag and organize the registry data elements in a controlled vocabulary.
The aim was to enable efficient user-selection of various stratifications of registry patients.
This required developing an automatic data cleaning and loading pipeline (using the R language) to provide easy integration of new data from the PMSIR into PMS_DN over time with consistent variable coding.
To facilitate further analysis by other researchers, variables representing clinical items in the survey were categorized as being either "historical" or "evolving".
"Historical" variables are immutable (e.g. birth weight), whereas "evolving" variables change over time (e.g. current weight).
On PMSIR, when a patient family retakes a survey, the previous entry is displayed to prevent erasure or duplication of earlier data.
Thus, we considered changing an "evolving" variable as updating it with the current value, and changing an "historical" variable as correcting a previous mistake.
All "evolving" data was stored for each time point, and the most recent entry for "historical" items was considered the most accurate (Supplemental data 1a and 1b).
This guarantees that the most recent record carries the most up-to-date "historical" information while storing the "evolving" parameters histories.

By reusing pre-existing genetic test results, we encountered a huge disparity in data collection techniques, accuracy, and references.
To circumvent this problem, we manually reviewed all genetic test results and extracted the coordinates and genome assembly of reported chromosomal abnormalities.
In addition, a trained geneticist specializing in the PMS further reviewed and confirmed genetic results.
Chromosomal coordinates for aCGH were extracted from the relevant structured fields (chromosome, gain/loss, start, end), from ISCN nomenclature, and from comments, when necessary.
When multiple assays surveying the same region were available, we choose for further analysis the most recent or most precise.
Specifically, we chose sequencing and SNP arrays over aCGH.
Using the UCSC LiftOver tool, chromosomal coordinates were transformed from their original human reference genome to the most recent assembly, GRCh38/hg38.[@rosenbloom_ucsc_2015]
Coordinates that did not map to GRCh38/hg38 were not used in the analysis.
We entered only terminal deletions in the subsequent analyses to ensure comparability and relevant use and interpretations of our models.
Contiguous deletions that were separated by a gap less than 5% of their total length were merged into one deletion to prevent underestimating their effect.

In the multi-Phenome-Wide Association study (PheWas [@denny_phewas:_2010]) reported here, we compared the extent of deleted genetic material on the terminus of chromosome 22 with the occurrence of each phenotypic feature, equating the extent of each deletion with the genetic anomaly of interest.
To this effect, we treated patients with *SHANK3* mutations as patients with terminal deletions going from the end of the chromosome up to the starting position of *SHANK3*.
We selected for analysis both physical and behavioral phenotypic features, including those commonly associated with PMS.
In order to satisfy the technical requirements of the regression models, we chose features with at least 5 respondents for each option.

We built multivariate models that associated phenotypic outcome with the deletion size, adjusting for the age and gender of patients.
When the outcome was a continuous variable (birth length, weight, head circumference and APGAR scores), we used linear regression.
When the outcome was binary (phenotype present or not), we utilized logistic regression.
Note: this method was used in previous deletion analyses of PMS.[@sarasua_clinical_2014 ; @sarasua_association_2011 ; @sarasua_22q13.2q13.32_2014]
For ordinal outcomes (severity of phenotype based on a four-point scale from “absent” to “always present,” and age ranges for the acquisition of developmental milestones), we used proportional odds logistic regression.[@valenta_proportional_2006]
We used the Benjamini-Hochberg procedure for False Discovery Rate (FDR) globally to adjust the p values for multiple hypotheses testing.[@benjamini_controlling_1995]
Odds ratios (and their 95% confidence intervals) are given for every million deleted base pairs.

Statistical analyses were conducted using R version `r str_c(version$major, version$minor, sep = ".")`.
The proportional odds logistic regression used the "clm" function from the "ordinal" package.
The project was approved by Harvard Medical School Institutional Review Board (HMS IRB14-2161).

# Results

## Population and variable selection

As of November 11 2018, date of the latest registry data export used in this study, 849 families (461 in the USA), consented to participate in the network and PCORnet studies.
`r article$nb_pat$demo` out of these 849 patients in the PMS_DN entered demographic information.
Among these `r article$nb_pat$demo` patients, `r article$nb_pat$clin` completed the clinical questionnaire and `r article$nb_pat$dev` the developmental questionnaire.
A total of `r article$nb_pat$pheno` patients had demographic and either clinical or developmental information entered.

Demographic information for the included patients is described in Table 1.
The selected population was significantly younger than the rest of the PMS_DN cohort, and less diverse with a majority of caucasian patients.
There were no demographic biases at inclusion regarding gender.
Age was compared using the non-parametric Mann & Whitney test, gender and race were compared using the Fisher test.
Completion rates ranged from `r prettyNum(article$completion["Min."] * 100, digits = 2)`% to `r prettyNum(article$completion["Max."] * 100, digits = 2)`% of items, with a mean completion rate of `r prettyNum(article$completion["Mean"] * 100, digits = 2)`%.

Chromosomal abnormalities span all chromosomes.
From the test results of `r article$nb_pat$all_gen` patients we extracted `r article$genetics$all` validated deletion/duplication ranges expressed relative to GRCh38/hg38 on the chromosome 22: `r article$genetics$X22$mut` mutations, `r article$genetics$X22$del` deletions, and `r article$genetics$X22$dup` duplications.
`r article$nb_pat$term_del` patients presented terminal deletion of the chromosome 22, including `r article$nb_pat$mut_sel` patients with point mutations of *SHANK3*.
Deletion sizes ranged between `r prettyNum(article$genetics$range["Min."]/10^3, digits = 2)` kb to `r prettyNum(article$genetics$range["Max."]/10^6, digits = 2)` Mb, with a median deletion size of `r prettyNum(article$genetics$range["Median"]/10^6, digits = 2)` Mb.

A total of `r article$nb_pat$included` patients who entered demographic and clinical or developmental phenotypic information, as well as presented terminal deletion of chromosome 22 or point mutation of *SHANK3*, were included for analysis.

Of the 1,300 patient-reported outcome (PRO) items present in the registry, `r article$PRO$sel` were selected as representing phenotypic outcomes.
PRO items not selected were items not referring directly to a symptom or a condition, such as “Does the patient have a primary cardiologist?”, “How was the genetic test paid?”, “Has the patient had any of the following tests – Manometry testing”, etc.
`r article$PRO$incl` phenotypes satisfied the inclusion criteria and were included in the analyses.

## Deletion analyses results

`r article$nb_pat$all_gen` patients had at least one genetic test result for which deletion/duplication coordinates were available or could be translated into the GRCh38/hg38 human genome assembly on chromosome 22.
Figure 1 represents the terminal abnormalities on chromosome 22 for these `r article$nb_pat$all_gen` patients.
Each patient's status is represented on one line, in order of their leftmost abnormality.
Duplicated regions are represented as blue boxes, deleted regions as red boxes, and mutations of *SHANK3* as asterisks.
The range of the *SHANK3* gene is represented as a grey vertical box across the figure.
Horizontal black lines help relate different chromosomal abnormalities for patients with more than one abnormality.

`r article$PRO$incl` phenotypes were tested against deletion sizes for `r article$nb_pat$included` patients using the models described in the methods section.
After correction of the p values, `r article$results_ranges %>% mutate(p.adj = p.adj %>% as.numeric) %>% filter(p.adj < 0.05) %>% nrow` phenotypes were significantly associated with deletions of greater size, meaning that these phenotypes are probably associated with at least one other gene than *SHANK3* on the long arm of chromosome 22, independently or in association with *SHANK3*.
The complete analysis results are available in Supplemental data 2, with accompanying figures for all groups of phenotypes in Supplemental data 3.

Classical morphological features described in the literature are associated with greater deletion sizes :
hyper-growth (p = `r pvalue("Too tall for age")`),
bushy eyebrows (p = `r pvalue("Bushy eyebrows")`),
large fleshy hands (p = `r pvalue("Large fleshy hands")`),
sacral dimple (p = `r pvalue("Sacral dimple")`),
and dysplastic toenails (p = `r pvalue("Dysplastic or unusual toenails")`) and fingernails (p = `r pvalue("Dysplastic or unusual fingernails")`).
Swelling of the extremities, for the most part the low limb `r result_for_group("Swelling")` was also encountered only in patients with the largest deletions.

All renal conditions `r result_for_group("Renal conditions")` are strongly associated with the deletion size, with most affected patients having only the largest deletions (Figure 2). Other renal symptoms, except for persistent fevers related to the recurrent urinary tract infections, were not associated with the deletion size.

Major delays in nearly all gross motor milestones `r result_for_group("Gross motor milestones")` are strongly and linearly associated with greater deletion sizes. They are shown in Figure 3.
These delays in gross motor milestones are accompanied with a range of conditions related to muscular tone: hypotonia (p = `r pvalue("Hypotonia")`),
hypo-reflexia (p = `r pvalue("Hypo-reflexia")`),
apnea (p = `r pvalue("Apnea")`),
fatigue (p = `r pvalue("Fatigues easily")`),
weak muscles (p = `r pvalue("Weak muscles")`),
poor reflexes (p = `r pvalue("Poor reflexes")`),
and feeding problems (p = `r pvalue("Feeding problems")`).

Feeding difficulties `r result_for_group("Feeding difficulties")`, necessitated naso-gastric tubes (p = `r pvalue("Nasogastric (NG) tube")`), and are not explained by feeding disorders (only PICA (p = `r pvalue("Pica")`) is associated, negatively, with the deletion size).

Only the presence of a heart murmur (p = `r pvalue("Heart murmur")`) and the broad category of congenital heart diseases (p = `r pvalue("Congenital heart disease")`), of which only the patent ductus arteriosus (p = `r pvalue("Patent Ductus Arteriosus")`) could be used in the analysis, were associated with bigger deletions.

Some behaviors such as daytime teeth grinding (p = `r pvalue("Daytime teeth grinding")`).

All the significantly associated phenotypes with positive effect sizes are presented in Table 2, with their respective raw p value, FDR adjusted p value, and odds ratio with 95% confidence interval (odds ratios are expressed for every million base pair deleted).
The presence of these features and symptoms is associated with greater deletion sizes amongst the patients, and so are likely due to the deletion of some upstream gene on chromosome 22.

> TODO :
> Abnormal Cranial CAT scan (p = `r pvalue("Abnormal Cranial CAT scan")`),
> Abnormal MRI (p = `r pvalue("Abnormal MRI")`),
> Irregular Ventricles (p = `r pvalue("Irregular Ventricles")`),
> Decreased Grey Matter (p = `r pvalue("Decreased Grey Matter")`),
> and
> Febrile seizures (p = `r pvalue("Febrile seizures")`).

Some anxiety disorders `r result_for_group("Anxiety disorders")` are negatively associated with the deletion size.
Attention disorder symptoms `r result_for_group("Attention disorder")` are also less present in patients with greater deletion sizes.
Nearly all the cognitive development surveyed `r result_for_group("Cognitive development")` are significantly impaired by the size of the deletion.

Basic communication milestones `r result_for_group("Communication milestones")` are acquired later than normal in patients with longer deletions, primitive non-verbal skils `r result_for_group("Communication – Non-verbal skills")` are rarer, and verbal speech `r result_for_group("Verbal speech")` is heavily impaired for patients with even small deletions.
However, syndromic etiologies `r result_for_group("Communication-related symptoms")`, or auditory processing disorder (p = `r pvalue("Auditory processing disorder")`) cannot explain these communication development delays, as they are negatively associated with the deletion sizes.
Hearing problems (apart from the installation of ear tubes (p = `r pvalue("Ear tubes")`) are not associated with the genetic status.

We observe delays in socio-emotional development, in all activities related to socialisation `r result_for_group("Socio-emotional development")`.

Sleep disturbances `r result_for_group("Sleep disturbances")` are less frequent in patients with greater deletions.

The detail of all the significantly associated phenotypes with negative effect sizes are presented in Table 3.
The absence of these features, behaviors or abilities is associated with greater deletion sizes amongst the patients, and so are likely due to the deletion of some upstream gene on chromosome 22.

> TODO:
> and differential diagnoses:
> ADHD (p = `r pvalue("ADHD")`),
> Pervasive Developmental Disorder (p = `r pvalue("Pervasive Developmental Disorder")`),
> and
> Autism (p = `r pvalue("Autism")`)
> and
> Autistic-like features (p = `r pvalue("Autistic-like features")`).

There were no associations with any neurological, orthopedic, nose and throat, thyroid, or allergy-related conditions.
Beside a mild association with chronic constipation (p = `r pvalue("Chronic constipation")`, OR = `r or("Chronic constipation")`), and aspiration (p = `r pvalue("Aspiration")`), no gastrointestinal conditions or symptoms were associated with the genetic defects.
Responses to sounds, movement, touch, oral input, or temperature variations were not associated with greater deletions.
Tolerance to pain was not associated either, as well as pain levels around anesthesia.
There were no associations with any vaccine reactions.

# TODO
> Abdominal bloating
> C-section
> learning disabilities
> ?Other classical dysmorphic features in PMS were not found associated with deletion size, and only mild forms of aggressive behavior.

# Discussion

We demonstrate the feasibility of using registry data based on patient-reported outcome questionnaires to conduct a multi-phenotype-genotype association study and further demonstrate replication of findings of previous prospective association studies, as well as discovery of novel findings in renal malformations and gross motor development.
This is the largest study of its kind in PMS and the first using the Phelan-McDermid Syndrome International Registry.
Using the PMSIR allowed us to recruit the largest sample size (`r article$nb_pat$included` patients) for a genotype-phenotype association analysis in PMS (Soorya et al.(2013), 32 patients [@soorya_prospective_2013], Sarasua et al.(2014), 70 patients [@sarasua_22q13.2q13.32_2014], Sarasua et al.(2011), 71 patients [@sarasua_association_2011], Sarasua et al.(2014), 201 patients [@sarasua_clinical_2014]).
The data provided by this patient-driven network differs from data obtained from medical records and is rich with information that only caregivers and families can provide.
The information being reported by the parents themselves, as proxy of the patients, allows the registry items to be more numerous and more specific in all domains, which would be very difficult and costly in a conventional academic-institution-led prospective study.
Indeed, children with PMS exhibiting hyperactive and aggressive behaviors, can be difficult to manage, and filling in extensive surveys during interviews with the pediatrician is very difficult, and subject to recall bias.
Parents have more time to fill the surveys at home when the child is asleep, they find it easier to make corrections to entered data, and are more likely to answer items more thoroughly and keep the surveys updated.
The consistency of the results throughout the different analyses and with previous findings demonstrates the reliability of the information contained in the registry.
The extended precision of the results shows the potential of analyses on the data from the PMSIR to pinpoint very specific phenotype associations in this syndrome.

Renal malformations, present in nearly 30% of PMS patients [@kolevzon_phelan-mcdermid_2014], are strongly associated with greater deletion sizes, with the distribution of deletion sizes hinting at a region of interest on chromosome 22, which had only been reported once for all renal abnormalities regardless of the precise condition [@soorya_prospective_2013].
This indicates that renal malformations in PMS, absent from patients with isolated *SHANK3* mutations [@durand_mutations_2007 ; @moessner_contribution_2007 ; @bozdagi_haploinsufficiency_2010 ; @macedoni-luksic_deletion_2013 ; @bonaglia_disruption_2001 ; @delahaye_chromosome_2009], but found in patients with interstitial deletions not involving *SHANK3* [@vittoria_disciglio_interstitial_2014], are related to different genes than *SHANK3* playing a role in the renal morphogenesis.

Greater deletion sizes are consistently and proportionally positively associated with gross motor development delays, as well as phenotypes related to muscle tone in the early stages of development, and feeding difficulties.
Some of these findings had been uncovered separately before [@sarasua_clinical_2014 ; @sarasua_association_2011 ; @sarasua_22q13.2q13.32_2014] but not consistently for each gross motor milestone, as previous studies only captured age at walk [@sarasua_clinical_2014 ; @sarasua_22q13.2q13.32_2014] or age at walk and crawl [@sarasua_association_2011].
Some of these features are found in PMS patients with interstitial deletions not affecting *SHANK3* [@wilson_interstitial_2008 ; @vittoria_disciglio_interstitial_2014].
They are also found in a lesser extent and less severity in PMS patients with isolated *SHANK3* mutations [@macedoni-luksic_deletion_2013 ; @bonaglia_disruption_2001 ; @delahaye_chromosome_2009].
The fact that all these tone-related conditions start to appear even with the smallest deletions, with the severity increasing proportionally to the deletion size suggests a cumulative role of multiple genes on the long arm of chromosome 22, including *SHANK3.*
The previously unreported associations of deletion sizes with gastrointestinal aspiration symptoms and apnea could also be attributed of this muscle tone severity gradient, and not to specific gastrointestinal or respiratory conditions.

Previous results of association with large fleshy hands and dysplastic toenails [@sarasua_clinical_2014 ; @sarasua_association_2011 ; @sarasua_22q13.2q13.32_2014] could be reproduced in this study, as well as results of increased prevalence of lymphedema and swelling of extremities [@soorya_prospective_2013] and aggressive behaviors [@sarasua_22q13.2q13.32_2014].

Common differential diagnoses of ADHD and ASD, as well as symptoms of both these disorders classes, are negatively associated with deletion size, meaning that patients with greater deletion sizes are less likely to present these symptoms or of having been misdiagnosed with one of these conditions.
This has previously been suggested [@sarasua_association_2011 ; @sarasua_22q13.2q13.32_2014], for ASD, to be caused by "masking" of autistic symptoms by the depth of the disabilities due to the greater deletion size.
Negative association with ADHD and hyperactivity symptoms can be linked with the spectrum of low muscle tone associated with greater deletion size.

Abnormal IRM and CAT scans, known in PMS patients [@soorya_prospective_2013] are found to be associated with greater deletion sizes, with reports of abnormal ventricles and decreased grey matter.
No other neurological morphological abnormalities or symptoms were associated with greater deletion sizes (such as the presence of an arachnoid cyst, motion sickness, or sensitivity to light).

Social, cognitive and communication development are inversely associated with deletion size, characterizing in detail the depth and breadth of intellectual disability [@leblond_meta-analysis_2014 ; @wilson_interstitial_2008].
Mostly abilities related to understanding social conventions and interacting with others seem to be related to greater deletions, whereas individual abilities and anxiety are evenly distributed throughout the sample.

This method of analysis has previously been considered as a proxy for the number of deleted genes and severity of the genetic condition [@soorya_prospective_2013].
However, if it can indicate the participation of other genes in the upstream part of chromosome 22 in relation to *SHANK3*, it cannot identify such genes or discard the implication of *SHANK3* in the presence of the phenotypes.
When used as a predictor of developmental delays and depth of cognitive or social disabilities, it can suggest cumulative effect of the interplay of multiple gene alterations.
While it does not come as a surprise that more deleted genetic material would cause more severe phenotypes, the consistency of associations only for gross motor milestones delays, and not with other groups of phenotypes such as cardiovascular, dental, endocrine, fine motor skills milestones, etc. indicates that a number of genes in this region of interest spanning 9 Mb at the end of chromosome 22 could be specifically responsible for gross motor functions and muscle tone.

Two different mechanisms can cause the risk augmentation for a phenotype with the deletion of a gene:

* haploinsufficiency, as with *SHANK3* and PMS, where the deletion of only one copy causes the phenotype.
* the "unmasking" of the effect of a pathological allele for an autosomic recessive trait.

The latter seems to be the principal mechanism, as there are patients with very large terminal deletion sizes who do not present some of the phenotypes found to be associated with greater deletion sizes.

Two main scenarios exist for those phenotypes that are not associated with greater deletion sizes: either they are not associated with PMS, and their prevalence in the sample is similar to the one in the general population (this would be the case for, for example, Pain urinating, Night terrors, Enuresis, or Congenital heart disease); or they are part of the core 22q13 deletion syndrome caused by the deletion of *SHANK3* and are present in nearly all of the patients in the sample.
This is the case for Intellectual disability, Severe adverse reaction to any vaccines, Teeth grinding, Joint laxity, or High pain tolerance level.

When used as a predictor for binomial outcomes such as the occurrence of a clearly defined somatic phenotype, use of the deletion can only act as a proxy for responsibility of chromosomal regions of interest and hint at potential breakpoints.
Comparisons using individual genes could help narrow down ranges of genes that might play a role in the phenotype, but this needs specific model developments as gene statuses inferred from this data would generate strong collinearity between contiguous genes affected in the same deletion ranges.

It is to be noted that PMS being caused by 22q13 deletions, a majority of patients are tested only for genetic defects on chromosome 22, specifically using array CGH or FISH techniques and providing ranges of deletions.
This is a bias in the data as most information is for chromosome 22 defects only, ignoring possible additional chromosomal aberrations.
The assumption in this analysis is that other potential defects are randomly distributed and don't affect the outcome.

A third source of data consisting of the natural language processing of the patients' clinical notes, encoded as UMLS concepts, will provide us with a range of phenotypes that were not captured by the registry questionnaires and will permit even more detailed analyses.

# Conclusions

In this study using data from the Phelan-McDermid Syndrome International Registry, we proceeded to demonstrate the viability of this source of data to conduct genotype-phenotype analyses.
We begin by providing a method of data curation and integration dealing with the shortcomings of the registry, and which should serve as a reference for future projects on this data.
We show that deletion size is associated with gross motor development, a number of diverse conditions painting a picture of a spectrum of generalized muscle tone impairments, with novel associations previously unreported.
We confirm the previous findings of association between greater deletion sizes and large fleshy hands, dysplastic toenails, sacral dimple, verbal speech ability, and socio-emotional development.

# Supplemental Data description

Supplemental Data contains two figures describing the data management workflow, one table of full results from the statistical analyses, and `r article$plots %>% length` figures detailing results of the analyses by group of features.

# Acknowledgments

The Phelan-McDermid Syndrome Foundation, the patients and their families. Chris Botka and the Harvard Medical School Research Computing center.

This work was partially funded through a Patient-Centered Outcomes Research Institute (PCORI) Award (PPRN-1306-04814) phase I and II for development of the National Patient-Centered Clinical Research Network, known as PCORnet; and by Research Grand EDU_R_FY2015_Q2_HarvardMedicalSchool_Avillach-NEW from Amazon Inc. National Institutes of Health − RFA-HG-13-009 − Centers of Excellence for Big Data Computing in the Biomedical Sciences (U54) − Grant Number 1U54HG007963-01.

The statements presented in this article are solely the responsibility of the author(s) and do not necessarily represent the views of the Patient-Centered Outcomes Research Institute (PCORI), its Board of Governors or Methodology Committee or other participants in PCORnet.

# Web Resources

All the code for ETL and analysis is openly available on github (https://github.com/hms-dbmi/PMSDN_R-ETL, https://github.com/hms-dbmi/PMSDN_GenoPheno)

# Figures titles and legends

## Figure 1

**Title**: Genetic status on chromosome 22 for all patients with a verified genetic test result.
**Legend**: Deletions (in red), *SHANK3* mutations (asterisks), and duplications (in blue) on the long arm of chromosome 22 (as shown on top), ordered by decreasing maximum extent of deleted genetic material, one line for each patient

## Figure 2

**Title**: Association between terminal deletion size and phenotypic status for renal abnormalities.
**Legend**: On the left: terminal deletions (in red) and *SHANK3* mutations (asterisks) for the included patients, ordered by decreasing size of terminal deletion size. On the right: status for each patient for every renal condition, lined up with the respective genetic status.

## Figure 3

**Title**: Association between terminal deletion size and phenotypic status for the age of acquisition of gross motor milestones.
**Legend**: On the left: terminal deletions (in red) and *SHANK3* mutations (asterisks) for the included patients, ordered by decreasing size of terminal deletion size. On the right: status for each patient for every developmental gross motor phenotype, lined up with the respective genetic status.

# Tables

```{r table1}
pander(article$table1,
       justify = c("left", rep("center", 9), "right"),
       caption = "Sample characteristics and comparison with the excluded patients",
       missing = "")
```


```{r table2}
article$results_ranges %>%
  rename(Phenotype = text) %>%
  filter(or > 1,
         p.adj < 0.05) %>%
  arrange(desc(or)) %>%
  select(Phenotype, p, p.adj, or, IC_OR) %>%
  mutate_all(funs(prettyNum(., digits = 3))) %>%
  as.data.frame %>%
  pander(split.table = Inf,
         caption = "Significantly associated phenotypes with largest positive effect sizes",
         justify = c("left", rep("center", 4)))
```


```{r table3}
article$results_ranges %>%
  rename(Phenotype = text) %>%
  filter(or < 1,
         p.adj < 0.05) %>%
  arrange(or) %>%
  select(Phenotype, p, p.adj, or, IC_OR) %>%
  mutate_each(funs(prettyNum(., digits = 3))) %>%
  as.data.frame %>%
  pander(split.table = Inf,
         caption = "Significantly associated phenotypes with largest negative effect sizes",
         justify = c("left", rep("center", 4)))
```


# References
