---
title: "Identification of genotype-phenotype associations in Phelan-McDermid Syndrome using patient-sourced data from an international registry"
keep_md: true
output:
  pdf_document:
    template: template.tex
    keep_tex: true
    latex_engine: xelatex
  word_document:
    reference_docx: template.docx
bibliography: PMS_DN_biblatex.bib
csl: humangenet.csl
author: Maxime Wack, Claire Hassen-Khodja, Michael T. McDuffie, Geraldine Bliss, Liz Horn, Andria Cornell, Cartik Kothari, Catalina Betancur, Megan O'Boyle, Isaac Kohane, Paul Avillach
abstract: |
  # **Abstract**
  Phelan-McDermid Syndrome (PMS) is a syndromic form of autism caused by terminal deletions of the long arm of chromosome 22 affecting at least the SHANK3 gene. It variably associates autism, global developmental delay, delayed speech, neonatal hypotonia, and mildly dysmorphic features. Isolated haploinsufficiency of SHANK3 has been shown to be responsible of a subset of PMS features. The PMS International Registry (PMSIR) compiles clinical data in the form of patient-reported outcomes, as well as patient-sourced genetic test results. Data from the PMSIR have been harmonized and integrated into the i2b2/tranSMART clinical & genomics data warehouse. We conducted genotype-phenotype analyses using regression models associating the deletion size as a predictor of the different clinical outcomes. `r article$nb_pat$included` patients were included, with deletion sizes ranging from `r article$genetics$range["Min."]/1000` kb to `r article$genetics$range["Max."]/10^6` Mb, with `r article$nb_pat$mut_sel` patients presenting small isolated SHANK3 mutations. Increased deletion size is significantly associated with delay in gross motor acquisitions, renal malformations, mild dysmorphic features (large fleshy hands, dysplastic toenails and sacral dimple), and a spectrum of conditions related to poor muscle tone, suggesting the implication of genes upstream of SHANK3. In this study using data from the PMSIR, we demonstrate the use of entirely patient-sourced registry data consisting of PRO items filled by the parents, and curated genetic test reports to conduct genotype-phenotype analyses. Known results are replicated and novel findings show the ability of registry data to uncover new associations between comorbidities and deleted chromosomal regions in PMS.
---

```{r load, echo = F, message = F}
library(rmarkdown)
library(knitr)
library(dplyr)
library(pander)
library(stringr)

load("article.Rdata")

pvalue <- function(var)
{
  article$results_ranges %>%
    filter(text == var) %>%
    .$p.adj %>%
    prettyNum(digits = 3)
}
opts_chunk$set(echo = F,
               message = F,
               error = F,
               warning = F,
               fig.width = 12,
               fig.height = 7)
```

# Introduction

The 22q13 deletion syndrome (OMIM 606232), also called Phelan-McDermid Syndrome (PMS) [@phelan_phelan-mcdermid_1993 ; @phelan_22q13.3_2012], is a syndromic form of Autism Spectrum Disorder (ASD) [@phelan_22q13.3_2012]. It associates autistic features, with rates of autism or autistic-like behavior ranging from 44% to 94% [@kolevzon_phelan-mcdermid_2014 ; @costales_phelanmcdermid_2015], and an estimate of 84% of cases using the gold standard diagnostic instrument, the Autism Diagnostic Observation Schedule [@soorya_prospective_2013]; global developmental delay in more than 98% of patients [@phelan_22q13.3_2012 ; @phelan_deletion_2008]; and Intellectual Disability (ID) with delayed or absent speech in nearly 100% of cases [@phelan_22q13.3_2012 ; @kolevzon_phelan-mcdermid_2014 ; @costales_phelanmcdermid_2015 ; @soorya_prospective_2013 ; @phelan_deletion_2008 ; @durand_mutations_2007 ; @leblond_meta-analysis_2014]. It accounts for an estimated 0.5% of ASD cases [@kolevzon_phelan-mcdermid_2014 ; @costales_phelanmcdermid_2015 ; @leblond_meta-analysis_2014 ; @gong_high_2012 ; @betancur_shank3_2013] and 2% of ID cases [@kolevzon_phelan-mcdermid_2014 ; @leblond_meta-analysis_2014 ; @gong_high_2012 @cooper_copy_2011], with an equal number of male and female patients [@costales_phelanmcdermid_2015 ; @phelan_deletion_2008].

In addition to ASD and ID, PMS patients are characterized with specific behavioral features, such as impairments in communication and social activities, chewing of non-food items (80-90%), teeth grinding (~25%) and aggressive behavior (10-15%) [@phelan_phelan-mcdermid_1993 ; @phelan_22q13.3_2012 ; @kolevzon_phelan-mcdermid_2014 ; @phelan_deletion_2008 ; @sarasua_clinical_2014]. The syndrome is a contiguous gene disorder caused by random deletions of the distal long arm of chromosome 22, deleting the *SHANK3* gene located at the end of the chromosome. The disease is diagnosed by genetic testing for deletion of the 22q13 chromosomal region. As of today, there are approximately 1,300 diagnosed cases worldwide (PMSF - The Phelan-McDermid Syndrome Foundation), with at least 520 cases confirmed in the US, and no treatment are yet available, although IGF1 has been shown to restore the *SHANK3* pathway[@shcheglovitov_shank3_2013] and has been used successfully in a phase 2 randomized controlled trial[@kolevzon_pilot_2014].

*SHANK3* (SH3 and multiple ankyrin repeat domain 3) is a gene coding for a post-synaptic scaffolding protein binding to several different neurotransmitter receptors and is considered to be of great importance in the assembly and plasticity of the postsynaptic density (PSD) in brain synapses. It has been shown to be involved in non-PMS autistic spectrum disorders [@durand_mutations_2007 ; @leblond_meta-analysis_2014 ; @betancur_shank3_2013 ; @shao_commonly_2014 ; @moessner_contribution_2007 ; @boccuto_prevalence_2013 ; @uchino_shank3_2013 ; @herbert_shank3_2011], further indicating its role in the neurobehavioral features of PMS. Deficiency (deletion or loss of function) of a single copy of *SHANK3* has been shown to be sufficient to cause the disease [@wilson_molecular_2003], in patient cases and mice models involving only *de novo* point mutations, or interstitial deletions only disrupting *SHANK3* [@durand_mutations_2007 ; @moessner_contribution_2007 ; @bozdagi_haploinsufficiency_2010 ; @macedoni-luksic_deletion_2013 ; @bonaglia_disruption_2001 ; @delahaye_chromosome_2009]. PMS patients with isolated *SHANK3* mutations or deletions show major speech delays and profound ID, global developmental delay, ASD symptoms, slightly delayed motor milestones, mild hypotonia, and minor dysmorphic features (dolichocephaly, epicanthic folds). *SHANK3* haploinsufficiency, with Fragile X Syndrome, are the two most frequent single gene genetic causes of autism/autism spectrum disorder.

Patients exhibit heterogeneous chromosomal abnormalities in the 22q13 region: terminal deletions, interstitial deletions, duplications and ring chromosome, resulting in randomly distributed deletions sizes from 100 kb to over 9 Mb [@phelan_deletion_2008]. Previous studies have shown that there is a gradient of severity in cognitive impairments [@leblond_meta-analysis_2014], associated with the size of the deletions, also linked to the presence of other phenotypic features [@soorya_prospective_2013 ; @sarasua_clinical_2014 ; @sarasua_association_2011 ; @sarasua_22q13.2q13.32_2014]. However, a growing number of cases reported shared features with the diagnosis of 22q13 deletion syndrome without presenting the classical terminal 22q deletion present in the majority of PMS cases. Instead, they present interstitial deletions with sizes as small as 0.72Mb beyond the range of the *SHANK3* gene, indicating that a subset of features of PMS are not caused by *SHANK3* haploinsufficiency alone [@fujita_girl_2000 ; @wilson_interstitial_2008 ; @simenson_patient_2014 ; @vittoria_disciglio_interstitial_2014 ; @maitz_expanding_2008]. These patients show severe developmental delays, hypotonia with feeding difficulties, overgrowth, speech impairments, seizures and abnormal neuroimaging, strabismus, congenital heart defects, kidney anomalies, ASD, and minor dysmorphic features (epicanthic folds, dysplastic toenails, fleshy hands).

The Phelan-McDermid Syndrome International Registry (PMSIR) [@PMSIR_phelan-mcdermid_2011], a patients’ family-powered registry was launched in 2011 by the Phelan-McDermid Syndrome Foundation, founded by the parents of PMS patients. It is the only international registry for this disease, and today includes more than half the worldwide population of known patients. The phenotypic and genotypic information contained in the registry is obtained and annotated by the patients’ parents.

We hypothesize that the retrospective, patient-reported data, including genetic test results obtained from the Phelan-McDermid Syndrome International Registry [@PMSIR_phelan-mcdermid_2011] could be used to conduct deletion analyses and uncover phenotypes and comorbidities significantly linked to genes other than *SHANK3*.

# Methods

The mission of the Phelan-McDermid Syndrome Data Network (PMS_DN) project, funded by the US Patient Centered Outcome Research Institute (PCORI) is to advance knowledge of Phelan-McDermid Syndrome and related conditions by integrating 1) Patient-reported outcomes from a patient registry, 2) Curated genetic test results, in the centralized i2b2/tranSMART platform to facilitate patient-centered research, and 3) clinical knowledge extracted from clinical notes. The latter was not used in this study. The network promotes a culture of transparency and authentic engagement and leadership of families in network vision, governance, and generation of research priorities. PMS_DN uses data entered on the online Phelan-McDermid Syndrome International Registry by the patients' parents. The registry comprises 1,300 patient-reported outcome (PRO) items over three distinct questionnaires, a “clinical” questionnaire with questions regarding diagnosed comorbidities, symptoms, tests, and treatments for the whole range of known pathologies and features associated with PMS; a “developmental” questionnaire, focusing on physical, motor, behavioral, cognitive, and social development; and an “adult” questionnaire with specific questions aimed at patients aged 15 or more, regarding the evolution of symptoms after puberty. The latter was not used in this study. The patients' families can retake the surveys for the phenotypic data any number of times, allowing for longitudinal analysis and ensuring the most complete information for each patient. Alongside phenotypic data, the registry contains curated genetic testing of chromosomal abnormalities from karyotypes, FISH probes, comparative genomic hybridization (aCGH), microarrays, and sequencing represented using the ISCN Standard Nomenclature [@international_standing_committee_on_human_cytogenetic_nomenclature_iscn_2013]. The scanned paper reports of the genetic test results are uploaded by the families on the platform then manually curated by trained genetic counselors, filling 57 structured fields to represent the genetic abnormalities.

All the phenotypic and genotypic data in the PMS registry were integrated in the i2b2/tranSMART[@athey_transmart:_2013 ; @canuel_translational_2015 ; @murphy_serving_2010] data warehouse to facilitate their integration with future updates of the registry and additional sources of data. In addition, a PMS ontology was created to tag the registry data elements in a controlled vocabulary and organizing them to allow for efficient user-selection of various stratifications of the patients in the registry. This required developing an automatic data cleaning and loading pipeline using the R language to provide easy integration of new data from the PMSIR into PMS_DN over time, and consistency in the way every variable is coded in the system. Variables representing each clinical item in the survey were categorized as being either historical or evolving over time, to facilitate further analysis by other researchers. All historical data was kept for each time point, and the most recent entry for historical items was kept as the most accurate value (Supplemental data 1a and 1b). This has enabled greater reliability in data entry. For example, when taking the survey at a new time point, displaying the previous entry prevents erasure or duplication of previously entered data, guaranteeing that the most recent record holds up to date information.

Because we reused already existing genetic test results, they presented a huge disparity in techniques, accuracy, and references. All the curated genetic test results for each patient were manually reviewed to extract the coordinates and genome assembly of the chromosomal abnormalities. Chromosomal coordinates for aCGH were extracted from the relevant structured fields (chromosome, gain/loss, start, end), from ISCN nomenclature, and from comments when necessary. When multiple assays were available for the same region, the most recent or most precise one was kept. In decreasing order of preference based on precision, we chose sequencing and SNP arrays over array CGH. Chromosomal coordinates were transformed from each original human genome assembly to the latest one available at the time of this study, GRCh38/hg38[@rosenbloom_ucsc_2015], using the UCSC liftOver tool. Coordinates that did not map to GRCh38/hg38 were kept in the original genome assembly and were not used in the analysis. Only the terminal deletions were kept in the analyses to ensure comparability and relevant use of the models and their interpretation.

For the analysis reported here, we compared the extent of deleted genetic material on the terminus of chromosome 22 with the occurrence of each phenotypic feature, in a multi-phenotype-genotype association study (aka Phenome-Wide Association study - PheWas [@denny_phewas:_2010]), using the deletion extent as the genetic anomaly of interest. Phenotypic features, both physical and behavioral, including features commonly associated with the diagnosis of PMS, were selected for analysis. Only features with strictly more than five patients in each subgroup were selected to satisfy technical requirements of the regression models. The comparisons were made using only data from patients for which deletion coordinates were available or translatable to GRCh38/hg38 human genome assembly, to ensure comparability and consistency of the results. We built multivariate models associating the phenotypic outcome with the deletion size, adjusted for the age and gender of the patients. Linear regression was used when the outcome was a continuous variable (birth length, weight, head circumference and APGAR scores), logistic regression when the outcome was binary (phenotype present or not), a method used in previous deletion analyses in PMS [@sarasua_clinical_2014 ; @sarasua_association_2011 ; @sarasua_22q13.2q13.32_2014], and proportional odds logistic regression [@valenta_proportional_2006] for ordinal outcomes (severity of a phenotype on a 4 point scale from “absent” to “always present”, age ranges for the acquisition of developmental milestones). The False Discovery Rate (FDR) method was used to adjust the p values for multiple hypotheses testing [@benjamini_controlling_1995]. Odds ratios (and their 95% confidence intervals) are given for every million deleted base pairs.

Statistical analyses were conducted using R version `r str_c(version$major, version$minor, sep = ".")`. The proportional odds logistic regression used the "clm" function from the "ordinal" package. The project was approved by Harvard Medical School Institutional Review Board (HMS IRB14-2161).

# Results

## Population and variable selection

As of October 12 2016, date of the latest registry data export used in this study, 623 families (334 in the USA), consented to participate in the network and PCORnet studies. `r article$nb_pat$demo` out of these 623 patients in the PMS_DN entered demographic information. Among these `r article$nb_pat$demo` patients, `r article$nb_pat$clin` completed the clinical questionnaire and `r article$nb_pat$dev` the developmental questionnaire. A total of `r article$nb_pat$pheno` patients had demographic and either clinical or developmental information entered. Demographic information for the included patients is described in Table 1. There were no demographic biases at inclusion regarding age or gender. Ages were compared using a non-parametric Mann & Whitney test, genders were compared using a Fisher test. Completion rates ranged from `r article$completion["Min."] * 100`% to `r article$completion["Max."] * 100`% of items, with a mean completion rate of `r article$completion["Mean"] * 100`%.

Chromosomal abnormalities span all chromosomes except 14, 17, and 18. From the test results of `r article$nb_pat$all_gen` patients we extracted `r article$genetics$all` validated deletion/duplication ranges expressed relative to GRCh38/hg38, including `r article$genetics$X22$del + article$genetics$X22$dup + article$genetics$X22$mut` on chromosome 22: `r article$genetics$X22$mut` mutations, `r article$genetics$X22$del` deletions, and `r article$genetics$X22$dup` duplications. `r article$nb_pat$term_del` patients presented terminal deletion of chromosome 22, including `r article$nb_pat$mut_sel` patients with point mutations of *SHANK3*. Deletion sizes ranged between `r article$genetics$range["Min."]/10^3` kb to `r article$genetics$range["Max."]/10^6` Mb, with a median deletion size of `r article$genetics$range["Median"]/10^6` Mb.

A total of `r article$nb_pat$included` patients who entered demographic and clinical or developmental phenotypic information as well as presented terminal deletion of chromosome 22 or point mutation of *SHANK3* were included for analysis.

Of the 1,300 patient-reported outcome (PRO) items present in the registry, `r article$PRO$sel` were selected as representing phenotypic outcomes. PRO items not selected were items not referring directly to a symptom or a condition, such as “Does the patient have a primary cardiologist?”, “How was the genetic test paid?”, “Has the patient had any of the following tests – Manometry testing”, etc. `r article$PRO$incl` phenotypes satisfied the inclusion criteria and were included in the analyses.

## Deletion analyses results

`r article$nb_pat$all_gen` patients had at least one genetic test result for which deletion/duplication coordinates could be translated into the GRCh38/hg38 human genome assembly on chromosome 22. Figure 1 represents the chromosomal abnormalities on chromosome 22 for these `r article$nb_pat$all_gen` patients. Panel A of the figure is a representation of chromosome 22 with the zoomed region framed in red. Panel B represents the deletion sizes for each patient, one line per patient, ordered by the decreasing total size of deleted genetic material. Duplicated regions are represented as blue boxes, deleted regions as red boxes, and mutations of *SHANK3* as dark red boxes. The range of the *SHANK3* gene is represented as a gray transparent box across the figure. Horizontal black lines help relate different chromosomal abnormalities for patients with more than one abnormality.

`r article$PRO$incl` phenotypes were tested against deletion sizes for `r article$nb_pat$included` patients using the models described in the methods section. After correction of the p values using the FDR method, `r article$results_ranges %>% mutate(p.adj = p.adj %>% as.numeric) %>% filter(p.adj < 0.05) %>% nrow` phenotypes were significantly associated with deletions of greater size, meaning that these phenotypes are probably associated with at least one other gene than *SHANK3* on the long arm of chromosome 22, independently or in association with *SHANK3*. Complete results are presented in Supplemental data 2, and figures for all groups of phenotypes in Supplemental data 3.

Significantly associated phenotypes with positive effect sizes are presented in Table 2, with their respective raw p value, FDR adjusted p value, and odds ratio with 95% confidence interval (odds ratios are expressed for every million base pair deleted). They include the age of acquisition for all main gross motor developmental milestones: (Figure 1)
Hold head (p = `r pvalue("Hold head up on his/her own")`),
Crawl (p = `r pvalue("Crawl on hands and knees")`),
Sit (p = `r pvalue("Sit when placed")`),
Walk (p = `r pvalue("Walk unassisted")`),
and
Climb stairs (p = `r pvalue("Climb stairs standing up without help")`)
where greater deletion sizes are associated with later ages of acquisition;
certain congenital dysmorphic kidney ailments: (Figure 2)
Vesicoureteral reflux (p = `r pvalue("Vesicoureteral Reflux ")`),
Hydronephrosis (p = `r pvalue("Hydronephrosis ")`),
Increased kidney size (p = `r pvalue("Increased kidney size")`),
Polycystic kidney (p = `r pvalue("Polycystic kidney")`),
Persistent fever related to kidney issues (p = `r pvalue("Persistent fever related to kidney issues")`),
and
Dysplastic kidney (p = `r pvalue("Dysplastic kidney")`);
a range of phenotypes related to gross motor and muscle tone:
Floppy baby (p = `r pvalue("Floppy baby")`),
Neonatal hypotonia (p = `r pvalue("Hypotonia")`),
Fatigues easily (p = `r pvalue("Fatigues easily")`),
Apnea (p = `r pvalue("Apnea ")`),
and
Fatigue (p = `r pvalue("Fatigues easily")`);
multiple items related to feeding difficulties during the first-year development:
Feeding problems (p = `r pvalue("Feeding problems")`),
Swallowing problems (p = `r pvalue("Had swallowing problems")`),
Required special feeds (p = `r pvalue("Required special feeds")`),
and
Poor suck (p = `r pvalue("Had poor suck")`);
dysmorphic features commonly described in PMS:
Too tall for age (p = `r pvalue("Too tall for age ")`),
Sacral dimple (p = `r pvalue("Does the patient have a sacral dimple?")`),
Large fleshy hands (p = `r pvalue("Large fleshy hands")`),
and
Dysplastic toenails (p = `r pvalue("Dysplastic or unusual toenails")`);
lymphatic system issues:
swelling of left (p = `r pvalue("Left foot")`),
and
right feet (p = `r pvalue("Right foot")`),
and
Primary Lymphedema (p = `r pvalue("Primary Lymphedema")`);
and central nervous system anomalies:
Abnormal Cranial CAT scan (p = `r pvalue("Abnormal Cranial CAT scan Test result")`),
Abnormal MRI (p = `r pvalue("Abnormal MRI Test result")`),
Irregular Ventricles (p = `r pvalue("Irregular Ventricles")`),
Decreased Grey Matter (p = `r pvalue("Decreased Grey Matter")`),
and
Febrile (p = `r pvalue("Has the patient ever had febrile seizures?")`)
and
non-febrile seizures (p = `r pvalue("Has the patient ever had non-febrile seizures?")`).
The presence of these features and symptoms is associated with greater deletion sizes amongst the patients, and so are likely due to the deletion of some upstream gene on chromosome 22.

Significantly associated phenotypes with negative effect sizes are presented in Table 3. They include behaviors related to hyperactivity:
Act as if driven by a motor (p = `r pvalue("Act as if driven by a motor")`),
Run around and climb excessively (p = `r pvalue("Run around or climb excessively when inappropriate")`),
and
Symptoms of ADD/ADHD (p = `r pvalue("Symptoms/diagnostic of ADD/ADHD")`);
social behaviors:
Plays alongside others but not with them (p = `r pvalue("Plays alongside others but not with them")`),
Engage in pretend play (p = `r pvalue("Engage in pretend play")`),
Respond to others emotions (p = `r pvalue("Respond to others emotions (e g  pats someone who is upset/crying) ")`),
Respond affectionately to caregivers (p = `r pvalue("Respond affectionately to caregivers ")`),
Hug/kiss caregivers, dolls, stuffed animals (p = `r pvalue("Hug/kiss caregivers, dolls, stuffed animals")`),
and
Shows awareness of imaginary characters (p = `r pvalue("Shows awareness of imaginary characters ")`);
cognitive development:
Understanding of the use of familiar objects (p = `r pvalue("Understanding of the use of familiar objects ")`),
Imitate household activities during play (p = `r pvalue("Imitate household activities during play ")`),
Use functional toys appropriately (p = `r pvalue("Use functional toys appropriately ")`),
Act out recent experiences using gestures, etc. (p = `r pvalue("Act out recent experiences using gestures, etc.")`),
Understand difference between food and objects (p = `r pvalue("Understand difference between food and objects ")`),
and
Observe actions and imitate them later (p = `r pvalue("Observe actions and imitate them later")`);
communication development:
Ability to follow directions (p = `r pvalue("Ability to follow directions")`),
Receptive language disorder (p = `r pvalue("Receptive language disorder")`),
Expressive language disorder (p = `r pvalue("Expressive language disorder")`),
and
Use of sign language (p = `r pvalue("Sign language")`);
and differential diagnoses:
ADHD (p = `r pvalue("ADHD")`),
Pervasive Developmental Disorder (p = `r pvalue("Pervasive Developmental Disorder ")`),
and
Autism (p = `r pvalue("Autism")`)
and
Autistic-like features (p = `r pvalue("Autistic-like features")`).
The absence of these features, behaviors or abilities is associated with greater deletion sizes amongst the patients, and so are likely due to the deletion of some upstream gene on chromosome 22.

No allergy, anesthesia related, cardiovascular, ears nose and throat, immunologic, psychiatric, sensory, skin, and sleep conditions were significantly associated with greater deletion sizes. Other classical dysmorphic features in PMS were not found associated with deletion size, nor forms of aggressive behavior. Non-febrile seizures were not found associated with deletion size, as opposed to febrile seizures. No fine motor developmental milestones were found associated with deletion size, when all major gross motor milestones were found to be associated with greater deletions.

# Discussion

We demonstrate the feasibility of using registry data based on patient-reported outcome questionnaires to conduct a multi-phenotype-genotype association study and further demonstrate replication of findings of previous prospective association studies, as well as discovery of novel findings in renal malformations and gross motor development. This is one of the largest study of its kind in PMS and the first using the Phelan-McDermid Syndrome International Registry. Using the PMSIR allowed us to recruit the second largest sample size for a genotype-phenotype association analysis in PMS (Soorya et al.(2013), 32 patients [@soorya_prospective_2013], Sarasua et al.(2014), 70 patients [@sarasua_22q13.2q13.32_2014], Sarasua et al.(2011), 71 patients [@sarasua_association_2011], Sarasua et al.(2014), 201 patients [@sarasua_clinical_2014]). The data provided by this patient-driven network differs from data obtained from medical records and is rich with information that only caregivers and families can provide. The information being reported by the parents themselves, as proxy of the patients, allows the registry items to be more numerous and more specific in all domains, which would be very difficult and costly in a conventional academic-institution-led prospective study. Indeed, children with PMS exhibiting hyperactive and aggressive behaviors, can be difficult to manage, and filling in extensive surveys during interviews with the pediatrician is very difficult, and subject to recall bias. Parents have more time to fill the surveys at home when the child is asleep, they find it easier to make corrections to entered data, and are more likely to answer items more thoroughly and keep the surveys updated. The consistency of the results throughout the different analyses and with previous findings demonstrates the reliability of the information contained in the registry. The extended precision of the results shows the potential of analyses on the data from the PMSIR to pinpoint very specific phenotype associations in this syndrome.

Renal malformations (hydronephrosis and vesicoureteral reflux), present in nearly 30% of PMS patients [@kolevzon_phelan-mcdermid_2014], are strongly associated with greater deletion sizes, with the distribution of deletion sizes hinting at a region of interest on chromosome 22, which had only been reported once for all renal abnormalities regardless of the precise condition [@soorya_prospective_2013]. This indicates that renal malformations in PMS, absent from patients with isolated *SHANK3* mutations [@durand_mutations_2007 ; @moessner_contribution_2007 ; @bozdagi_haploinsufficiency_2010 ; @macedoni-luksic_deletion_2013 ; @bonaglia_disruption_2001 ; @delahaye_chromosome_2009], but found in patients with interstitial deletions not involving *SHANK3* [@vittoria_disciglio_interstitial_2014], are related to different genes than *SHANK3* playing a role in the renal morphogenesis.

In addition to cognitive and social development delays and depth of intellectual disability [@leblond_meta-analysis_2014 ; @wilson_interstitial_2008], greater deletion sizes are consistently and proportionally positively associated with gross motor development delays, specifically and differentially from fine motor acquisitions, as well as phenotypes related to muscle tone in the early stages of development, and feeding difficulties, and negatively associated with diagnosis and symptoms related to hyperactivity. Some of these findings had been uncovered separately before [@sarasua_clinical_2014 ; @sarasua_association_2011 ; @sarasua_22q13.2q13.32_2014] but not consistently for each gross motor milestone, as previous studies only captured age at walk [@sarasua_clinical_2014 ; @sarasua_22q13.2q13.32_2014] or age at walk and crawl [@sarasua_association_2011]. Some of these features are found in PMS patients with interstitial deletions not affecting *SHANK3* [@wilson_interstitial_2008 ; @vittoria_disciglio_interstitial_2014]. They are also found in a lesser extent and less severity in PMS patients with isolated *SHANK3* mutations [@macedoni-luksic_deletion_2013 ; @bonaglia_disruption_2001 ; @delahaye_chromosome_2009]. The fact that all these tone-related conditions start to appear even with the smallest deletions, with the severity increasing proportionally to the deletion size suggests a cumulative role of multiple genes on the long arm of chromosome 22, including *SHANK3.* The previously unreported associations of deletion sizes with gastrointestinal aspiration symptoms and apnea could also be part of this muscle tone severity gradient.

Previous results of association with large fleshy hands and dysplastic toenails/fingernails [@sarasua_clinical_2014 ; @sarasua_association_2011 ; @sarasua_22q13.2q13.32_2014] could be reproduced in this study, and results of increased prevalence of lymphedema [@soorya_prospective_2013] and aggressive behaviors [@sarasua_22q13.2q13.32_2014] were not replicated here.

This method of analysis has previously been considered as a proxy for the number of deleted genes and severity of the genetic condition [@soorya_prospective_2013]. However, if it can indicate the participation of other genes in the upstream part of chromosome 22 in relation to *SHANK3*, it cannot identify such genes or discard the implication of *SHANK3* in the presence of the phenotypes. When used as a predictor of developmental delays and depth of cognitive or social disabilities, it can suggest cumulative effect of the interplay of multiple gene alterations. While it does not come as a surprise that more deleted genetic material would cause more severe phenotypes, the consistency of associations only for gross motor milestones delays, and not with other groups of phenotypes such as cardiovascular, dental, endocrine, fine motor skills milestones, etc. indicates that a number of genes in this region of interest spanning 9 Mb at the end of chromosome 22 could be specifically responsible for gross motor functions and muscle tone.

Interestingly, if we compare phenotypic behaviors between (with all the limits that are needed) mice and men, in Mei et al [@mei_adult_2016] *SHANK3* KO mice show signs of decreased exploratory behaviors and social interaction, increased anxiety, and signs of repetitive grooming. This is concordant with our results; tested phenotypes related to anxiety and to decreased social interactions are all found to not be associated with greater deletion sizes, indicating the role of *SHANK3* deletion alone to be sufficient.

When used as a predictor for binomial outcomes such as the occurrence of a clearly defined somatic phenotype, use of the deletion can only act as a proxy for responsibility of chromosomal regions of interest and hint at potential breakpoints. Comparisons using individual genes could help narrow down ranges of genes that might play a role in the phenotype, but this needs specific model developments as gene statuses inferred from this data would generate strong collinearity between contiguous genes affected in the same deletion ranges.

It is to be noted that PMS being caused by 22q13 deletions, a majority of patients are tested only for genetic defects on chromosome 22, specifically using array CGH or FISH techniques and providing ranges of deletions. This is a strong bias in the data as most information is for chromosome 22 defects only, ignoring possible additional chromosomal aberrations.

A third source of data consisting of the natural language processing of the patients' clinical notes, encoded as UMLS concepts, will provide us with a range of phenotypes that were not captured by the registry questionnaires and will permit even more detailed analyses.

# Conclusions

In this study using data from the Phelan-McDermid Syndrome International Registry, we proceeded to demonstrate the viability of this source of data to conduct genotype-phenotype analyses. We begin by providing a method of data curation and integration dealing with the shortcomings of the registry, and which should serve as a reference for future projects on this data. We show that deletion size is associated with gross motor development, a number of diverse conditions painting a picture of a spectrum of generalized muscle tone impairments, with novel associations previously unreported. We confirm the previous findings of association between greater deletion sizes and large fleshy hands, dysplastic toenails, sacral dimple, verbal speech ability, and socio-emotional development.

# Supplemental Data description

Supplemental Data contains two figures describing the data management workflow, one table of full results from the statistical analyses, and 31 figures detailing results of the analyses by group of features.

# Acknowledgments

The Phelan-McDermid Syndrome Foundation, the patients and their families. Chris Botka and the Harvard Medical School Research Computing center.
This work was partially funded through a Patient-Centered Outcomes Research Institute (PCORI) Award (PPRN-1306-04814) phase I and II for development of the National Patient-Centered Clinical Research Network, known as PCORnet; and by Research Grand EDU_R_FY2015_Q2_HarvardMedicalSchool_Avillach-NEW from Amazon Inc. National Institutes of Health − RFA-HG-13-009 − Centers of Excellence for Big Data Computing in the Biomedical Sciences (U54) − Grant Number 1U54HG007963-01.  
The statements presented in this article are solely the responsibility of the author(s) and do not necessarily represent the views of the Patient-Centered Outcomes Research Institute (PCORI), its Board of Governors or Methodology Committee or other participants in PCORnet.

# Web Resources

All the code for ETL and analysis is openly available on github (https://github.com/hms-dbmi/ETL/tree/master/data_projects/pms_dn/pms_registry/R-ETL, https://github.com/hms-dbmi/Avillach-Lab/tree/master/PMS_DN/GenoPheno)

# Figures titles and legends

## Figure 1

**Title**: Genetic status on chromosome 22 for all patients with a verified genetic test result.  
**Legend**: Deletions (in red), *SHANK3* mutations (in dark red), and duplications (in blue) on the long arm of chromosome 22 (as shown in Panel A), ordered by decreasing size of total deleted genetic material, one line for each patient (Panel B)

## Figure 2

**Title**: Association between terminal deletion size and phenotypic status for renal abnormalities.  
**Legend**: On the left: terminal deletions (in red) and *SHANK3* mutations (in dark red) for the included patients, ordered by decreasing size of terminal deletion size. On the right: status for each patient for every renal phenotype, lined up with the respective genetic status.

## Figure 3

**Title**: Association between terminal deletion size and phenotypic status for the age of acquisition of gross motor milestones.  
**Legend**: On the left: terminal deletions (in red) and *SHANK3* mutations (in dark red) for the included patients, ordered by decreasing size of terminal deletion size. On the right: status for each patient for every developmental gross motor phenotype, lined up with the respective genetic status.

# Tables

```{r table1}
pander(article$table1,
       justify = c("left", rep("center", 6), "right"),
       caption = "Sample characteristics and comparison with the excluded patients", 
       missing = "")
```
  

```{r table2}
article$results_ranges %>%
  rename(Phenotype = text) %>%
  filter(or > 1,
         p.adj < 0.05) %>%
  arrange(desc(or)) %>%
  select(Phenotype, p, p.adj, or, IC_OR) %>%
  mutate_each(funs(prettyNum(., digits = 3))) %>%
  pander(split.table = Inf,
         caption = "Significantly associated phenotypes with largest positive effect sizes",
         justify = c("left", rep("center", 4)))
```
  

```{r table3}
article$results_ranges %>%
  rename(Phenotype = text) %>%
  filter(or < 1,
         p.adj < 0.05) %>%
  arrange(or) %>%
  select(Phenotype, p, p.adj, or, IC_OR) %>%
  mutate_each(funs(prettyNum(., digits = 3))) %>%
  pander(split.table = Inf,
         caption = "Significantly associated phenotypes with largest negative effect sizes",
         justify = c("left", rep("center", 4)))
```


# References


